woocommerce/woocommerce-ios
# Add background for button bar of order list for edge-to-edge look

Part of #3716 

# Description
@designsimply found while beta testing v7.2 that the order list is still not extending edge-to-edge in dark mode. This is due to the inconsistent background color in dark mode.

<img src="https://user-images.githubusercontent.com/1119271/128251048-7322607b-c115-4237-8ee9-f69ef886b440.PNG" />

# Changes
- Added new background view underneath the button bar view. This view's leading and trailing edges are constrained to its super view and not safe area.
- Set background color for the background view the same color as the button bar view's. This gives the illusion that the button bar view extends edge-to-edge.

# Screenshots
| Light Mode | Dark Mode |
| ------ | ------ |
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-08-05 at 09 48 17](https://user-images.githubusercontent.com/5533851/128283987-86f81f1d-5075-4365-a91f-ec0a127896c6.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-08-05 at 09 48 27](https://user-images.githubusercontent.com/5533851/128283989-65d6627e-e916-4d31-ac55-ca30db164fda.png) |

# Testing
1. Select Orders tab.
2. Switch to landscape mode. 
3. Notice that the screen expands edge-to-edge in both light and dark mode.

Update release notes:
